### DISCUSSION TITLE
[Pod kill] Post https://chaos-mesh-controller-manager.chaos-testing.svc:443/validate-auth?timeout=30s: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "chaos-mesh-ca

### DISCUSSION THREAD

**Discussion by zhidongzhang** [chaos-mesh/chaos-mesh#4132] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**
1.17.3

**What version of Chaos Mesh are you using?**
2.6.1  or 2.3.2

**What did you do? / Minimal Reproducible Example**
[Pod kill] error.api.internal_server_error: Internal error occurred: failed calling webhook "vauth.kb.io": Post https://chaos-mesh-controller-manager.chaos-testing.svc:443/validate-auth?timeout=30s: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "chaos-mesh-ca")

**What did you expect to see?**
Pod kill normal

**Comment by STRRL** [comment]
The webhook is not well configured, it's complex situation based on your setup.

please make sure your environment is clean and try to reinstall chaos mesh from helm.

related issue might be helpful: https://github.com/chaos-mesh/chaos-mesh/issues/918
